feature/about/src/main/java/io/github/droidkaigi/confsched2019/about/ui/item/AboutSection.kt

```diff
@@ -41,4 +44,21 @@ class AboutSection : Section() {
             )
         )
     }
+
+    private fun openVenueOnGoogleMap(context: Context) {
```

Comment:
Please move this method to ActivtyActionCreator. 
You can use it like this 🙏 
https://github.com/DroidKaigi/conference-app-2019/pull/253/files#diff-e940ede799d93174cb44acfc965faab8R14

Reply:
That pr is merged. Please merge or rebase to this branch from master 🙏